Source/WebCore/platform/graphics/gstreamer/GStreamerCommon.cpp

```diff
@@ -722,7 +722,11 @@ Ref<SharedBuffer> GstMappedOwnedBuffer::createSharedBuffer()
 
 GstMappedFrame::GstMappedFrame(GstBuffer* buffer, const GstVideoInfo* info, GstMapFlags flags)
 {
+#if GST_CHECK_VERSION(1, 20, 0)
     gst_video_frame_map(&m_frame, info, buffer, flags);
+#else
+    gst_video_frame_map(&m_frame, const_cast<GstVideoInfo*>(info), buffer, flags);
+#endif
```

Comment:
How about always doing the cast? A non-`const` variable can be always passed when a function takes a `const` argument:

```suggestion
    // Remove the cast once the minimum required GStreamer version is 1.20.0
    gst_video_frame_map(&m_frame, const_cast<GstVideoInfo*>(info), buffer, flags);
```

At least this would prevent the duplication.

Reply:
done